src/hyperbolic_system.hpp

```diff
@@ -227,91 +311,125 @@ void HyperbolicSystem<problem_t>::ReconstructStatesPPM(amrex::MultiFab const &q_
 		quokka::Array4View<amrex::Real, DIR> leftState(leftState_in[bx]);
 		quokka::Array4View<amrex::Real, DIR> rightState(rightState_in[bx]);
 
-		// permute array indices according to dir
-		auto [i, j, k] = quokka::reorderMultiIndex<DIR>(i_in, j_in, k_in);
+		HyperbolicSystem<problem_t>::template ReconstructStatesPPM<DIR>(q, leftState, rightState, n, i_in, j_in, k_in, iReadFrom, iWriteFrom);
+	});
+}
+
+template <typename problem_t>
+template <FluxDir DIR>
+AMREX_GPU_HOST_DEVICE void HyperbolicSystem<problem_t>::ReconstructStatesPPM(arrayconst_t &q_in, array_t &leftState_in, array_t &rightState_in,
+									     amrex::Box const &cellRange, amrex::Box const & /*interfaceRange*/,
+									     const int nvars, const int iReadFrom, const int iWriteFrom)
+{
+	BL_PROFILE("HyperbolicSystem::ReconstructStatesPPM(Arrays)");
```

Comment:
You can fix the warning by changing this line to:
`const BL_PROFILE("HyperbolicSystem::ReconstructStatesPPM(Arrays)");`

Reply:
Done.